foreign_cc/built_tools/pkgconfig_build.bzl

```diff
@@ -18,6 +22,9 @@ def _pkgconfig_tool_impl(ctx):
         "%s install" % make_data.path,
     ]
 
+    if os_name(ctx) == "macos":
+        script[0] = "CFLAGS=-Wno-int-conversion CXXFLAGS=-Wno-int-conversion ./configure --with-internal-glib --prefix=$$INSTALLDIR$$"
```

Comment:
The problem with this command is it completely overrides CFLAGS and CXXFLAGS which means the flags from the exec configuration toolchain will be ignored. I suspect this will be better done by a select in the BUILD file for pkgconfig that conditionally adds these as copts so that any other flags are maintained when building the pkg config tool.

Reply:
updated, not sure if it's the right direction.